test/prettier.mjs

```diff
@@ -27,64 +32,63 @@ const rule = eslintPluginPrettier.rules.prettier;
 // ------------------------------------------------------------------------------
 
 const eslint = new ESLint({
-  baseConfig: {
-    parserOptions: {
-      ecmaVersion: 'latest',
-      ecmaFeatures: {
-        jsx: true,
+  overrideConfigFile: true,
+  overrideConfig: [
+    {
+      languageOptions: {
+        parserOptions: {
+          ecmaVersion: 'latest',
+          ecmaFeatures: {
+            jsx: true,
+          },
+          sourceType: 'module',
+        },
       },
-      sourceType: 'module',
     },
-    extends: 'plugin:prettier/recommended',
-    overrides: [
-      {
-        // `.prettierignore` will be used by default which is unexpected for these test fixtures
-        files: 'test/fixtures/**/*',
-        rules: {
-          'prettier/prettier': [
-            'error',
-            {},
-            {
-              fileInfoOptions: {
-                ignorePath: '.eslintignore',
-              },
+    recommendedConfig,
+    // `.prettierignore` will be used by default which is unexpected for these test fixtures
+    {
+      files: ['test/fixtures/**/*'],
+      rules: {
+        'prettier/prettier': [
+          'error',
+          {},
+          {
+            fileInfoOptions: {
+              ignorePath: '.eslintignore',
             },
-          ],
-        },
-      },
-      {
-        files: ['*.html'],
-        parser: '@html-eslint/parser',
-      },
-      {
-        files: ['*.{md,mdx}'],
-        extends: 'plugin:mdx/recommended',
-        settings: {
-          'mdx/code-block': true,
-        },
-      },
-      {
-        files: '**/eslint-plugin-svelte3/*.svelte',
-        plugins: ['svelte3'],
```

Comment:
`eslint-plugin-svelte3` is deprecated https://github.com/sveltejs/eslint-plugin-svelte3?tab=readme-ov-file#%EF%B8%8F-please-use-eslint-plugin-svelte-instead